src/Cron/CronExpression.php

```diff
@@ -73,6 +73,25 @@ public static function factory($expression, FieldFactory $fieldFactory = null)
     }
 
     /**
+     * Validate a CronExpression.
+     *
+     * @param string $expression The CRON expression to validate.
+     *
+     * @return bool True if a valid CRON expression was passed. False if not.
+     * @see Cron\CronExpression::factory
+     */
+    public static function isValidExpression($expression = '')
```

Comment:
Oh, sorry, force of habit to indicate the variable type of the argument. I can remove it if you want.